docs/install.md

```diff
@@ -86,7 +86,23 @@ Building from source requires:
 
 This is an advanced approach that most people will not require.
 
-To install the prebuilt libvips binaries from a custom URL,
+To install the prebuilt libvips binaries from a GitHub releases mirror,
+set the `sharp_libvips_binary_host` npm config option
+or the `SHARP_LIBVIPS_BINARY_HOST` environment variable.
```

Comment:
What does the `SHARP_LIBVIPS_BINARY_HOST` environment variable provide beyond the logic being introduced by the (implicit) `npm_config_sharp_libvips_binary_host` environment variable?

Reply:
I guess the main rationale was just following along with the npm config / environment variable combo pattern that was used for the Sharp Dist Base URL. I've found personally that environment variables are slightly easier to work with in some build systems than npm config changes for some work I've done in the past so I guess that was also in the back of my mind..

Happy to go either way however so will await your decision and take it from there :)

Reply:
Thanks, let's keep things consistent with `npm_config_sharp_binary_host` and expose only `npm_config_sharp_libvips_binary_host`. After this is merged, I'd like to deprecate `SHARP_DIST_BASE_URL` at some point (and eventually remove it) as I think this new dynamic option you're proposing will help/allow people to upgrade more easily.